src/models/UserMapper.php

```diff
@@ -264,6 +301,11 @@ public function setTrustedStatus($trustedStatus, $user_id)
     }
 
 
+    /**
+     * @param int $user
+     * @return bool|string
```

Comment:
Please add a blank line after the ```@param```s and move the ```@return``` to the last line of the DocBlock